std/assembly/atomics.ts

```diff
@@ -0,0 +1,99 @@
+export namespace Atomics {
+  // @ts-ignore: decorator
+  @inline
+  export function load<T>(array: ArrayBufferView<T>, index: i32): T {
+    if (isFloat<T>()) ERROR("Atomics.load accept only integer shared typed arrays");
```

Comment:
Doesn't this error happen already when trying to compile `atomic.load<f32>` or similar?

Reply:
No. It just got:
```ts
ERROR AS100: Operation not supported.

 atomic.load<f32>(changetype<usize>(array.buffer) + (0 << alignof<f32>()) + array.byteOffset);
             ~~~
```

Reply:
I see, yeah. Have been thinking about changing `Operation not supported.` into `Operation not supported: $0` (with a reason), so perhaps we should do that instead so we can have better errors in these situations?

Reply:
Yeah. I guess better if this error throw earlier in `atomic.load`